src/processor/operator/aggregate/aggregate_hash_table.cpp

```diff
@@ -676,36 +676,31 @@ bool AggregateHashTable::compareEntryWithKeys(const uint8_t* keyValue, const uin
 
 compare_function_t AggregateHashTable::getCompareEntryWithKeysFunc(DataTypeID typeId) {
     switch (typeId) {
-    case INTERNAL_ID: {
+    case INTERNAL_ID:
```

Comment:
I think our code style is to have {} even each case is a one line return statement. So revert this change.

Reply:
reverted